src/lib/db.js

```diff
@@ -48,9 +50,15 @@ function createObservationsReplicationStream () {
   return observationsDb.log.replicate()
 }
 
+function getObservationTimestampStream (options) {
+  const opts = options || {}
+  return observationsTimestampIndex.getDocumentStream(opts)
```

Comment:
Don't forget to listen on `index.ready()` before returning the stream: https://github.com/noffle/osm-p2p-timestamp-index#indexreadydone